test/package.json

```diff
@@ -50,6 +50,7 @@
     "codechain-sdk": "^2.0.0-alpha.2",
     "codechain-stakeholder-sdk": "https://github.com/majecty/codechain-stakeholder-sdk-js#v2.0.0-alpha.1",
     "elliptic": "^6.4.1",
+    "foundry-rpc": "github:MSNTCS/foundry-rpc-js#v2.0.0-alpha.1",
```

Comment:
What do you think about creating a branch named "alpha" in the CodeChain-io/foundry-rpc repository.

Reply:
I followed the same name pattern that you used for `codechain-stakeholder-sdk-js`.

Reply:
The `codechain-stakeholder-sdk-js`'s version came from the CodeChain version. We are creating a new rpc project for Foundry. Since Foundry is not a stable yet, it does not have a version. So I thought that we don't need the version for foundry-rpc yet.

Reply:
Releasing a program with a version of greater than 1.0.0 ensures the users that the program is stable. Our foundry, however, is not stable yet.

Reply:
I will change it.